components/main.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+
+import { store } from "../store";
+import { Provider } from "react-redux";
+import Root from "../routes";
```

Comment:
Add the prettier plugin for sorting your imports.
Sorting should be next:
- libs (the first is React)
- pages/components
- utils
- constants
- images

Reply:
done